xblockutils/public/tabs.css

```diff
@@ -0,0 +1,115 @@
+#xblock-editor {
+	margin-bottom: 0;
+	height: 100%
+}
+
+#xblock-editor .editor_content_and_tabs {
+	width: 100%;
+	height: calc(100% - 55px)
+}
+
+#xblock-editor #editor_window_title {
+	float: left
+}
+
+#xblock-editor input[type=number] {
+	background-image: -moz-linear-gradient(#F2F2F2, #fff);
+	background-image: -webkit-linear-gradient(#F2F2F2, #fff);
+	background-image: linear, false, #F2F2F2, #fff;
+	width: 45%
+}
+
+#xblock-editor #editor_header {
+	background-color: #e5e5e5;
+	width: 100%;
+	top: 0;
+	height: 42px
+}
+
+#xblock-editor #editor_header .editor_tabs {
+	width: 35%;
+	min-width: 190px;
+	max-width: 360px;
+	float: right
```

Comment:
These widths and heights are quite brittle, e.g if the block title is long, or the tab names are long, they don't fit in the heading anymore, and can lead to weird issues with scrolling in the modal.

Reply:
That's right, @pomegranited. We should probably apply different styling for the tabs.

Reply:
Resolved by bd59138 .